### ISSUE TITLE
Configuration for model locations

### ISSUE THREAD

**Issue by buddha314** [allenai/scispacy#469]
Allows for abstract model locations away from AWS.  This file sets AWS as model source

https://github.com/allenai/scispacy/blob/main/scispacy/candidate_generation.py

For instance https://github.com/allenai/scispacy/blob/main/scispacy/candidate_generation.py#L43 

Allow these to be set by environment or configuration files.

**Comment by dakinggg** [comment]
Makes perfect sense @buddha314, I'd be happy to accept a PR doing this!

**Comment by buddha314** [comment]
Initial thinking is to take the root url as an environment variable with the AWS location as the default.

**Comment by dakinggg** [comment]
Makes sense, I might consider making a new `LocalLinkerPaths` that reads from environment variables.
